### PR TITLE
feat: add helper scripts sample

### PR DESCRIPTION
**Pull request by farbodahm**
Add sample script usages of DLZ helper functions.

### CODE REVIEW

**Review by rehanvdm — COMMENTED** [review]
Yes, let's just wait on confirmation about the "argument lifting" but I see there are a few things you can update so long:

1. Don't call it config_basic, but config_minimum to keep in line with the TS one and how it was before?
2.  Change the config_complete.py as well + update the README.md again
3. Create a central app.py again and then use the config_minimum.py

**Comment by farbodahm** [comment]
@rehanvdm Applied. But now we should make sure that JSII is producing the same signature that our TS constructs have.

**Comment by rehanvdm** [comment]
I'm on it

File: scripts/bootstrap.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from aws_data_landing_zone import Scripts
+from data_landing_zone_example_python import config_minimum
```
Comment (rehanvdm) [inline scripts/bootstrap.py]: Can you change these imports to have a similar format as in `app.py`

```python
from data_landing_zone_example_python.config_minimum import config
```

and then pass it
```
scripts.boostrap_all(config)
```

**Review by rehanvdm — APPROVED** [review]

**Pull request merged**